api/v1alpha1/releaselink_webhook_unit_test.go

```diff
@@ -0,0 +1,155 @@
+//
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func TestReleaseLinkCreateValidatingWebhook(t *testing.T) {
+
+	tests := []struct {
+		name  string
+		newRL ReleaseLink
```

Comment:
Could we use a more readable name?
```suggestion
		releaseLink ReleaseLink
```